PGConfig.py

```diff
@@ -26,6 +26,10 @@
 delimiters = [' ', '=', '<', '>'] # Delimiters for tokenizing log lines [list of single characters]
 force_branch = [] # Parser tree depths where all branches are generated for all tokens, starts with 0 and also counts delimiters [list of integers]
 force_var = [] # Parser tree depths where all tokens are merged to variable, starts with 0 and also counts delimiters [list of integers]
+merge_branches = False # Merge similar branches to one branch
```

Comment:
Please state the allowed data type in square brackets at the end of the parameter descriptions as for the other parameters.